src/test/ui/proc-macro/invalid-punct-ident-1.rs

```diff
@@ -3,6 +3,11 @@
 // FIXME https://github.com/rust-lang/rust/issues/59998
 // normalize-stderr-test "thread.*panicked.*proc_macro_server.rs.*\n" -> ""
 // normalize-stderr-test "note:.*RUST_BACKTRACE=1.*\n" -> ""
+// normalize-stderr-test "error: internal compiler error.*\n" -> ""
```

Comment:
So if I'm reading this right, we're now emitting an `ICE` message if a proc macro panics? We should not do that. People should not report broken proc macros on the rustc repo

Reply:
This isn't caused by broken proc macros, it's a legitimate ICE: https://github.com/rust-lang/rust/issues/59998

Reply:
To expand on this, there's a bug in rustc that causes the ICE to print, and it's triggered by certain cases of proc macros doing invalid things (that are documented to panic). I don't think it is very easy to trigger it, it requires doing something like this: https://github.com/rust-lang/rust/blob/61dced18277a8cd55c963502db5a4cdf837858f3/src/test/ui/proc-macro/auxiliary/invalid-punct-ident.rs

Reply:
Thanks for the explanation!